corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -480,6 +484,13 @@ def get_payload(self):
             normal_providers = get_restore_providers()
             for provider in normal_providers:
                 for element in provider.get_elements(self.restore_state):
+                    if element.tag == 'fixture' and len(element) == 0:
+                        # There is a bug on mobile versions prior to 2.27 where
+                        # a parsing error will cause mobile to ignore the element
+                        # after this one if this element is empty.
+                        # So we have to add a dummy empty_element child to prevent
+                        # this element from being empty.
+                        ElementTree.SubElement(element, 'empty_element')
```

Comment:
@ctsims @phillipm i realize this was the fix discussed in the ticket but can you confirm:

- that this won't cause any other issues by introducing this extra dummy child tag
- if this needs to be done for any other elements other than fixtures